### ISSUE TITLE
snappyDB size is too big

### ISSUE THREAD

**Issue by xujinyang** [nhachicha/SnappyDB#51]
libsnappydb-native.so in armeabi armeabi-v7 mips x86  is too big ,then l can'n config this

**Comment by xxxifan** [comment]
you can donwload the dist package and only put armeabi/armeabi-v7 libs into your project,it runs on most android phones.
And I hope the author can update the disk package, for I don't want to see the annoying debug logs and I dont want to add full of libs into my project :)

**Comment by nhachicha** [comment]
@xxxifan thx for pointing this, I just updated the dist package for `0.5.2` to use a release build https://github.com/nhachicha/SnappyDB/tree/master/dist

@xujinyang you can also use Gradle `productFlavors` to produce different APK per ABI 
```
android {
    ...

    flavorDimensions "abi", "version"

    productFlavors {
        freeapp {
            flavorDimension "version"
            ...
        }

        x86 {
            flavorDimension "abi"
            ...
        }
    }
}
```
Cheers,

**Comment by oba2cat3** [comment]
Still - the size of the Native libs is too large.
The cost of maintaining/testing multiple flavors as a solution is a big problem.

**Comment by huangcd** [comment]
Agree, the size is too large to use. And produce different APK per ABI seems only work for Google Play, not suitable for other app store.
